src/main/res/layout/export_progress_activity.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="vertical">
+
+    <include layout="@layout/toolbar" />
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_weight="1"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <TextView
+            android:id="@+id/export_progress_done"
+            style="@style/TextMedium"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="24dp"
+            android:layout_marginTop="48dp"
+            app:layout_constraintEnd_toStartOf="@id/export_progress_slash"
+            app:layout_constraintHorizontal_chainStyle="packed"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:text="1" />
+
+        <TextView
+            android:id="@+id/export_progress_slash"
+            style="@style/TextMedium"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/generic_separator_done_total"
+            app:layout_constraintBottom_toBottomOf="@id/export_progress_done"
+            app:layout_constraintEnd_toStartOf="@id/export_progress_total"
+            app:layout_constraintStart_toEndOf="@id/export_progress_done"
+            app:layout_constraintTop_toTopOf="@id/export_progress_done" />
+
+        <TextView
+            android:id="@+id/export_progress_total"
+            style="@style/TextMedium"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="24dp"
+            app:layout_constraintBottom_toBottomOf="@id/export_progress_slash"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@id/export_progress_slash"
+            app:layout_constraintTop_toTopOf="@id/export_progress_slash"
+            tools:text="10" />
+
+        <ProgressBar
```

Comment:
We can move the progressbar into the actionbar, then it looks very similar to the dialog: 
Never done this, just know that it recommened for Material design: https://material.io/components/progress-indicators

Reply:
Don't you think is a better option the progress bar in the middle of the screen in this case?

Read the "Placement" section in the link you point out: https://material.io/components/progress-indicators (also there is a video with an example and I'd do it like that video shows).

Reply:
Styling like Material Design says done...